app/views/landing/index.html.erb

```diff
@@ -1,6 +1,5 @@
 <div class="bg-neutral lg:w-3/4 mx-auto rounded-lg text-neutral-content">
   <div class="sm:px-[5em]">
-    <%= render_navbar %>
```

Comment:
This completely removes the the navbar from the landing page, right? Screenshots would be helpful here.

Reply:
ah, got it; just leverages the `layouts/_navbar` partial, no matter where you are in the app.